tests/DynatraceMetricExporter.spec.ts

```diff
@@ -60,13 +60,45 @@ describe("DynatraceMetricExporter", () => {
 describe("MetricExporter.export", () => {
 	beforeEach(() => nock.cleanAll());
 
+	function getCounterResourceMetric(
```

Comment:
Would it be possible to move these functions to the end? I find it gets in the way of reading the tests. We can do it in a follow up as well, not blocking.

Reply:
I always forget about this. :man_facepalming:. 
Done in [0ed1e82](https://github.com/dynatrace-oss/opentelemetry-metric-js/pull/33/commits/0ed1e82ac1b3a7d8d99de460471be8cf01ff0508), thanks for the reminder. :slightly_smiling_face: